dbal/dsn.go

```diff
@@ -1,6 +1,25 @@
 package dbal
 
+import (
+	"regexp"
+)
+
 const (
 	SQLiteInMemory       = "sqlite://file::memory:?_fk=true"
 	SQLiteSharedInMemory = "sqlite://file::memory:?_fk=true&cache=shared"
 )
+
+// SQLite can be written in different styles depending on the use case
+// - just in memory
+// - shared connection
+// - shared but unique in the same process
+// see: https://sqlite.org/inmemorydb.html
+func IsSqlite(dsn string) bool {
```

Comment:
This should instead be `IsMemorySQLite` right?

Reply:
You are right! Will change quick